pennylane/devices/qubit/simulate.py

```diff
@@ -24,7 +24,28 @@
 from .sampling import measure_with_samples
 
 
-def get_final_state(circuit, debugger=None):
+INTERFACE_TO_LIKE = {
```

Comment:
Can this be shared with the interface module?

Reply:
I wanted to make a standalone one here because the one in interfaces/execution.py is a mess, but I'll make a task for myself to clean this up in the future.